Replace PNGs with SVGs where applicable and provide PNGS as fallback

Volker-E opened the pull request:
Replacing PNGs with SVGs where applicable – at the same time
providing PNGs as fallback to old Android and IEs.
Also:
- Adding `role=img` for Safari/Webkit <= 8 on SVG `img` tags
- Removing non-branding “.” from Visual Style Principles images
- And fine-tuning one `alt` attribute content.

Bug: [T156531](https://phabricator.wikimedia.org/T156531)
Bug: [T192137](https://phabricator.wikimedia.org/T192137)

Volker-E commented:
**This is not yet production-ready as we're waiting for updated imagery and also need to fine-tune SVG export further.**

prtksxna commented on design-principles.html:
```diff
@@ -103,15 +103,15 @@ <h2>Open to collaboration</h2>
 						<p>In order to facilitate understanding it is important to rely on familiar concepts. Keeping consistency within our designs, and maintaining common patterns helps people relate quickly and orientate easily.</p>
 						<figure>
-							<img src="img/design-principles/consistency.png" alt="Screenshots of various Foundation software products with consistent user-interface attributes">
+							<img src="img/design-principles/consistency.png" onerror="this.src='img/design-principles/consistency.png'" alt="Screenshots of various Foundation software products with consistent user-interface attributes" role="img">
```
Missed adding the SVG here

Volker-E commented:
We're going for the resources we've got so far now and will update remaining continuously, in order to already provide mobile users best-possible experience and not to let this sit for months here.

Pl217 commented on design-principles.html:
```diff
@@ -126,15 +126,15 @@ <h2>Open to collaboration</h2>
 						<p>Wikimedia projects provide reliable educational knowledge. At the same time, our projects are a source of entertainment and satisfy human curiosity. This requires us to find the right balance: we need to convey trust with our solutions but still remain entertaining and fresh.</p>
 						<figure>
-							<img src="img/design-principles/trust.png" alt="Screenshot of trending readers topic graph">
+							<img src="img/design-principles/trustworthy-yet-joyful.svg" onerror="this.src='img/design-principles/trustworthy-yet-joyful.png'" alt="Screenshot of trending readers topic graph" role="img">
```
Fonts in this SVG are different from fonts used in PNG fallback. SVG fonts are less readable on my machine and less visually balanced because there is no bold font weight like on PNG.

Volker-E replied:
@Pl217 Would you provide a screenshot from your machine? Readability is important and would be a blocker to the idea that I'm following in the implementation here.

Volker-E commented:
@Pl217 Dismissing your review: Have addressed all issues addressable and difference between SVG and PNGs is fine from current imagery available. Will move this forward in future patches against T156531
